### ISSUE TITLE
tool/charts: versions overwriting each other

### ISSUE THREAD

**Issue by sh0rez** [grafana/tanka#508]
It is possible to `require` the same chart at different versions. However, the entry that comes last in the `requires` list effectively overwrites the others during `helm pull`.

Proposed solution would be a `output` or `name` key in `Requirement` to specify where the downloaded chart shall go. Also Tanka should catch this case during `chart add` and request you to specify the above new field.

**Comment by bcbrockway** [comment]
I was just starting to look into how we might waterfall chart versions down dev, qa, prod, etc. without having to put the libs into separate git repos and vendoring the charts in there. I even tried adding a `name` key into the chartfile.yaml in the hope it might work before finding this ticket :)

TLDR; we'd be really interested in this!
